### PR TITLE
Add BRZ token

### PR DESCRIPTION
**Pull request by cesariovlabs**
Adds BRZ token on RSK Mainnet and Testnet.

To force the backend to load the tokens, they were added first as _custom tokens_. Then the UI was able to recognize the tokens and show them.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -56,26 +57,26 @@ const config = {
...
     BTC: {
-      Mainnet: 'https://live.blockcypher.com/btc/tx',
-      Testnet: 'https://live.blockcypher.com/btc-testnet/tx',
+      Mainnet: 'https://live.blockcypher.com/btc/txs',
+      Testnet: 'https://live.blockcypher.com/btc-testnet/txs',
     },
     RBTC: {
-      Mainnet: 'https://explorer.rsk.co/tx',
-      Testnet: 'https://explorer.testnet.rsk.co/tx',
+      Mainnet: 'https://explorer.rsk.co/txs',
```
Comment (lucachaco) [inline config.js]: I dont have this change

Comment (ilanolkies) [reply config.js]: Let's review it in another PR. I reverted the change since I don't need it either

**Comment by bitcoinuser** [comment]
Wow, amazing that https://www.brztoken.io/ will be minted on Rsk blockchain.

**Comment by ilanolkies** [comment]
Waiting for https://github.com/rsksmart/rsk-contract-parser/pull/11

File: src/common/wallet/cointype.js [hunk 2]
```diff
@@ -184,7 +195,7 @@ const coinType = {
     type: 'Testnet',
     symbol: 'BRZ',
     contractAddress: ASSETS_CONTRACT.BRZ.Testnet,
-    precision: 18,
+    precision: 4,
```
Comment (ilanolkies) [inline src/common/wallet/cointype.js:198]: Fixed this too

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**